### ISSUE TITLE
Script Error and VS crash when opening any .md file

### ISSUE THREAD

**Issue by victor-david** [madskristensen/MarkdownEditor#111]
### Installed product versions
- Visual Studio: 2017 Community
- This extension: 1.11.201
- Windows 64 bit Ultimate

### Description
When opening any .md file, an error dialog (it's from IE I believe) pops up.

![markdown1](https://cloud.githubusercontent.com/assets/17145322/25306958/16329284-275e-11e7-9965-03b6c3721261.png)

then (re: mermaid.js)

![markdown2](https://cloud.githubusercontent.com/assets/17145322/25307003/ea960a56-275e-11e7-9538-4eef5fb6268c.png)

### Steps to recreate
1. Open an .md file in Visual Studio
2. Script Errror pops up from prism.js. Click Yes or No.
3. Same script error from mermaid.js. Click Yes or No.
4. Try to change some text in the .md file
5. Another script error "Prism is undefined". Click Yes or No.
6. Visual Studio crashes
 
If you stop after step 3 (don't try to change text, just close the .md), the VS doesn't crash.

### Current behavior
The extension currently does not work because these scripts don't get loaded.  And then VS crashes.

### Expected behavior
No script error. No VS crash.

**Comment by victor-david** [comment]
I don't have too many README,md's. But, for example, this one:

https://github.com/victor-david/robo-crypto/blob/master/README.md

or this one

https://github.com/victor-david/panama/blob/master/README.md

I also created an empty file *test.md*, added only one line with a "# TEST" - then tried to open it. Same thing.

Does that help? I may not have understood your question the way you intended...

Could this be related to the IE version installed? I (ahem) hate IE and so I just leave the version that gets installed with Win 7 the way it is and never use it. It's IE 8. I ask b/c that looks like an IE Script Error dialog to me.

**Comment by victor-david** [comment]
Yes, all markdown documents that I've tried do the same thing. Haven't been able to open anything successfully. I'll have to think about whether it's worth it or not to upgrade IE. I never use it and it seems that an IE upgrade usually drags along other unintended consequences. Hmmm... but that's probably the culprit.

**Comment by MV10** [comment]
Today I suddenly began seeing the same errors with VS2017 v15.5.2 and extension v1.11.214 on Win 10 Pro x64, IE 11.726.15063.0 (the latest, I guess, I don't use IE either) . 

The strange part is that it I installed it about four days ago, and it worked great that whole time, including many trouble-free hours writing (this)[https://github.com/MV10/mv10.github.io/blob/master/_posts/2017-12-23-easy-configuration-sharing-with-azure-key-vault.md] long blog post -- then immediately after I pushed the md to github, the extension began crashing like clockwork whenever I open *any* md file. However, I think that was a coincidence, I had other md files already pushed and it wasn't crashing on those.

**Comment by MV10** [comment]
Interesting, makes sense. Thanks for taking the time to check. 

By way of explanation: Github Pages use Jekyll pre-processing which allows those {% %} Liquid template tags you saw. I was using triple-tick code-fencing while writing, then removed them at the last minute to get Rouge code highlighting instead.

**Comment by Eibwen** [comment]
Just ran into this issue (Steps 1-4, repeat.  Don't have the crash issue)

Extension version 1.12.236, VS2017 professional 15.7.3

I ended the `.md` file with a style block:
```<style>
.class {
  /*styling*/
}
</style>```

If I put that inside of triple-tick and close then reload the file, it works fine again (the styling works fine after that as well, and worked fine when i originally added it)

I also have elements like `<span class="class">colored label</span>` that I don't have to remove/modify.  Only the `<style>` block that is at the end and on its own lines
